### PR TITLE
[ZH] Update `Rules`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Rules/zh.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ outdated_since: 2787f766bf4cf7b8e2e3afd824e15adcb4e79798
 
 禁言的时长随着违反次数的增加**自动增长**，管理员也不能控制你被禁言的时间。因此多次违反可能导致你被长期禁言。
 
-严重违反规定时的封禁时间最小值通常为48小时或更长，这类行为通常是违规图片以及骚扰其他玩家。
+严重违反规定时的封禁时间最小值通常为42小时或更长，这类行为通常是违规图片以及骚扰其他玩家。
```
Comment (heipizhu4) [inline wiki/Rules/zh.md:65]:
```suggestion
严重违反规定时的最短封禁时间通常为42小时，这类行为通常是违规图片以及骚扰其他玩家。
```

**Review by heipizhu4 — COMMENTED** [review]
Mostly good, please avoid translationese. “请避免翻译腔”。这句话或更长可以省略，因为最小值已经涵盖了意思。

File: wiki/Rules/zh.md (hunk 1 above)
Comment (YumeMuzi) [reply wiki/Rules/zh.md:65]: 改成 严重违反规定时，封禁时间通常最短为42小时 语句会通顺很多

Comment (FlyingFish23) [reply wiki/Rules/zh.md:65]: tbh I haven't changed anything but 48 to 42. all the rest is exact from before.

**Review by RushFTK — CHANGES_REQUESTED** [review]
seemed i forgot to submit changes?

**Comment by TPGPL** [comment]
@FlyingFish23 are you planning to continue working on #7613 and #7612 at any time?

**Pull request closed without merging**